PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--- Use a prefix like [TASK], [BUGFIX], [DOC] or [CGL] and provide a general summary of your changes in the Title above -->
```

Comment:
We didn't use such prefixes until now and I'm not too big of a fan of them. I'd short the sentence to simply
> \<!--- Provide a general summary of your changes in the Title above -->